src/protocols/chainsync.rs

```diff
@@ -396,7 +400,7 @@ pub fn parse_msg_roll_forward(cbor_array: Vec<Value>) -> Option<(BlockHeader, Ti
                                         }
                                         _ => {
                                             warn!("invalid cbor! code: 340");
-                                            return None
+                                            return None;
```

Comment:
This would better be just `None` rather than `return None;`. But don't worry, I can fix that.

Reply:
ok